AmplifyPlugins/Core/AWSPluginsCore/Model/Decorator/ModelIdDecorator.swift

```diff
@@ -11,10 +11,15 @@ import Amplify
 /// Decorate the GraphQLDocument with the value of `Model.Identifier` for a "delete" mutation or "get" query.
 public struct ModelIdDecorator: ModelBasedGraphQLDocumentDecorator {
 
-    private let id: Model.Identifier
+    private let input: ModelDecoratorInput
 
-    public init(id: Model.Identifier) {
-        self.id = id
```

Comment:
So I just realized that we're changing the public interface, so even though it's public from AWSPluginsCore, we still have to be careful to keep things backwards compatible. Developers could have been using these public decorators themselves to create a valid GraphQL document. we need to go back and figure out how we should address this